src/main/kotlin/net/atos/zac/app/admin/ZaakafhandelParametersRestService.kt

```diff
@@ -182,11 +183,17 @@ class ZaakafhandelParametersRestService @Inject constructor(
     fun listZaakbeeindigRedenenForZaaktype(
         @PathParam("zaaktypeUUID") zaaktypeUUID: UUID?
     ): List<RESTZaakbeeindigReden> =
+        mutableListOf(createHardcodedZaakbeeindigRedenen()).apply {
```

Comment:
I don't like these constructs with mutable list, is there no 'join' operation we can use?

Reply:
`add` does not work, but happily `+` does different thing